### DISCUSSION TITLE
How to store and track build metrics?

### DISCUSSION THREAD

**Discussion by fniephaus** [community/community#37656] [Actions]
Hi all,

We'd like to allow the users of our GitHub action to track some simple build metrics over time (e.g., scatter plots or PR diffs). For this, we'd essentially need to store a small JSON file per commit and a way to look up these files again for specific commits.

What would be the best way to implement something like this? The GHA cache and artifacts do not seem to be designed for this (please correct me if I'm wrong!), and otherwise, I can only think about hacky solutions like storing the JSON in a GitHub comment on the commit, which is not what we want.

Any pointers are appreciated. Thanks!

**Comment by ossanna16** [comment]
Hi there @fniephaus and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
